change/@microsoft-teams-js-0dc95483-5ebc-4037-a819-ab2a4968825e.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "add code comment for dialog",
```

Comment:
Please follow the guidance for change files (found here: https://github.com/OfficeDev/microsoft-teams-library-js/blob/main/CONTRIBUTING.md#change-log-using-beachball).

In this case, I suggest (note that 'dialog.submit' is encased in backticks that GitHub auto-formats):

Added clarifying comment to `dialog.submit` to indicate the dialog is closed when `submit` is called.